R/zchunk_LA119.solar.R

```diff
@@ -29,10 +29,37 @@ module_gcam.usa_LA119.Solar_DISABLED <- function(command, ...) {
     # Load required inputs
     states_subregions <- get_data(all_data, "gcam-usa/states_subregions")
     NREL_us_re_capacity_factors <- get_data(all_data, "gcam-usa/NREL_us_re_capacity_factors")
-    NREL_us_re_technical_potential <- get_data(all_data, "gcam-usa/NREL_us_re_technical_potential")
+    NREL_us_re_technical_potential <- get_data(all_data, "gcam-usa/NREL_us_re_technical_potential")   #This data is not being used in this chunk
+
+    # Perform computations: Create scalars to scale capacity factors read in in the assumptions file in the energy folder to adjust for varying irradiance by state.
+
+    NREL_us_re_capacity_factors %>% gather(fuel,average,-State) %>%
+      filter (State=="Average") %>%
+      select (-State) -> Average_fuel
+
+    NREL_us_re_capacity_factors %>%
+      gather(fuel,value,-State) %>%
+      filter (State != "Average") %>%
+      full_join(Average_fuel, by = "fuel") %>%
```

Comment:
@bpbond : I think I'm missing something. full_join_error_no_match doesn't seem to be a dplyr function.